tests/english_member_page_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/english_members_page.rb'
+
+describe EnglishMembersPage, :vcr do
+  describe 'member_urls' do
+    it 'should return the expected number of urls' do
+      url = 'http://www.althingi.is/altext/cv/en/'
+      EnglishMembersPage.new(response: Scraped::Request.new(url: url).response)
+                        .member_urls
+                        .count
+                        .must_equal 63
+    end
+  end
+
+  describe 'member_urls' do
```

Comment:
You have two `describe` blocks with the same name. If these need to be separate, you should give them different names, otherwise you should simply have separate `it` blocks within the one `describe` section.